person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :@name, :@age
```

Comment:
- This is how to write the getters and setters `attr_accessor :name, :age` let's fix this, and to improve readability, let's move it up before the constructor

Reply:
@zieeco Thanks for the review and your valuable feedback.